### PR TITLE
Update debug-live-azure-apps-troubleshooting.md

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**
Expand on the 401 unauth error with details to handle EasyAuth. 

Currently I reference a blog post for authorization.json details, but I have also created a GitHub issue (https://github.com/MicrosoftDocs/azure-docs/issues/54988) to update the official Microsoft docs. I'll update the link to the official MicrosoftDocs once it's available.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@WilliamXieMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@WilliamXieMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/debugger/debug-live-azure-apps-troubleshooting.md [hunk 1]
```diff
@@ -25,13 +25,37 @@ If you see the following error in the **Output** window during the attempt to at
...
+      "path_prefix": "/41C07CED-2E08-4609-9D9F-882468261608/api/agent",
+      "policies": {
+        "unauthenticated_action": "AllowAnonymous"
+      }
+    }
+  ]
+}
+```
+
+The first route effectively secures your app domain similar to **Log in with [IdentityProvider]**. The second route exposes the SnapshotDebugger AgentLaunch endpoint outside of authentication which does the pre-defined action of starting the SnapshotDebugger diagnostic agent _only if_ the SnapshotDebugger preinstalled site extension is enabled for your app service. Please see https://azure.github.io/AppService/2016/11/17/URL-Authorization-Rules.html for more details on the authorization.json configuration.
```
Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:57]:
```suggestion
The first route effectively secures your app domain similar to **Log in with [IdentityProvider]**. The second route exposes the SnapshotDebugger AgentLaunch endpoint outside of authentication, which performs the pre-defined action of starting the SnapshotDebugger diagnostic agent *only if* the SnapshotDebugger preinstalled site extension is enabled for your app service. For more details on the authorization.json configuration, please see [URL authorization rules](https://azure.github.io/AppService/2016/11/17/URL-Authorization-Rules.html).
```

**Comment by PRMerger9** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
#sign-off

**Pull request merged**